client/components/domains/registrant-extra-info/style.scss

```diff
@@ -31,6 +20,12 @@
 			margin: 0;
 		}
 	}
+
+	.form-label .form-label__optional {
```

Comment:
Quick note that we do need this `.form-label` here to be more specific than the default styling